crates/shared-wasm-class-groups/src/lib.rs

```diff
@@ -34,10 +34,16 @@ pub fn protocol_public_parameters() -> ProtocolPublicParameters {
     >(encryption_scheme_pp)
 }
 
-pub fn network_dkg_final_output() -> <Secp256k1Party as mpc::Party>::PublicOutput {
+pub fn network_dkg_final_output() -> Box<<Secp256k1Party as mpc::Party>::PublicOutput> {
     // Safe to unwrap as we're using a hardcoded constant.
-    let protocol_public_parameters = STANDARD.decode(&NETWORK_DKG_OUTPUT).unwrap();
-    bcs::from_bytes(&protocol_public_parameters).unwrap()
+    let protocol_public_parameters = base64::decode(&NETWORK_DKG_OUTPUT).unwrap();
```

Comment:
```suggestion
    let protocol_public_parameters = base64::decode(&NETWORK_DKG_OUTPUT).unwrap();
```
use base64::{engine::general_purpose::STANDARD, Engine as _};